packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -92,6 +100,14 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
     }
   }
 
+  private validateMaxAgeInMinutes(resultReuseConfigurationMaxAge?: cdk.Duration) {
+    if (resultReuseConfigurationMaxAge === undefined || cdk.Token.isUnresolved(resultReuseConfigurationMaxAge)) return;
+    const maxAgeInMinutes = resultReuseConfigurationMaxAge.toMinutes();
```

Comment:
```suggestion
    const maxAgeInMillis = resultReuseConfigurationMaxAge.toMilliseconds();
    if (maxAgeInMillis > 0 && maxAgeInMillis < cdk.Duration.minutes(1).toMilliseconds()) {
      throw new Error(`resultReuseConfigurationMaxAge must be between greater than or equal to 1 minute or 0, got ${maxAgeInMillis}ms`);
    }
    const maxAgeInMinutes = resultReuseConfigurationMaxAge.toMinutes();
```
We should also check for the property to be at least 1 minute when specified, otherwise `resultReuseConfigurationMaxAge: cdk.Duration.seconds(1)` will throw an unhandled exception.